core-java-modules/core-java-14/src/test/java/com/baeldung/java14/foreign/api/ForeignMemoryUnitTest.java

```diff
@@ -0,0 +1,91 @@
+package com.baeldung.java14.foreign.api;
+
+import jdk.incubator.foreign.*;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.BlockJUnit4ClassRunner;
+
+import java.lang.invoke.VarHandle;
+import java.nio.ByteOrder;
+
+import static org.hamcrest.core.Is.*;
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.*;
+import static org.junit.Assert.assertThat;
+
+public class ForeignMemoryUnitTest {
+
+    @Test
+    public void whenAValueIsSet_thenAccessTheValue() {
+        long value = 10;
+        MemoryAddress memoryAddress =
+                MemorySegment.allocateNative(8)
```

Comment:
Indentation guidelines prescribe a 4-space indent for nested blocks and 2-space indent for continuation lines. Please adjust the indentation in this file accordingly.

Reply:
Thanks for the review and the feedback. I've incorporated the changes as asked. Kindly review,